addons/survey/static/src/js/survey.js

```diff
@@ -145,7 +145,9 @@ openerp.website.if_dom_contains('.js_surveyform', function (the_form) {
     // });
 
     // Launch prefilling
-    prefill();
+    if(prefill_controller){
+        prefill();
+    }
```

Comment:
Why this changes ?
I think the commit fix your problem (if there was one) : https://github.com/odoo/odoo/commit/11f147f2d94c8aff729590183e24337b2c076c86